package.json

```diff
@@ -49,7 +49,9 @@
   "dependencies": {
     "@ethersproject/abi": "^5.5.0",
     "@ethersproject/bytes": "^5.5.0",
+    "@hashgraph/proto": "^2.9.0",
     "dotenv": "^10.0.0",
+    "isomorphic-unfetch": "^3.1.0",
```

Comment:
These are both used by the new `HederaMirrorNetwork` class to fetch and deserialize the REST API network responses into valid Hashgprah SDK objects.